### DISCUSSION TITLE
关于上午/下午课程安排、操作等的个人建议

### DISCUSSION THREAD

**Discussion by DarkModest** [Class-Widgets/Class-Widgets#27] [建议]
- "XX-上午（下午）"的课程表编辑有些麻烦。建议将其改为双栏，左栏为上午课表，右栏为下午课表的形式。
- 建议添加一个模板课表模式：每天的早读、大课间、晚自习等时间固定，仅需修改相关课程即可。
- 触摸屏适配，尤其是在使用时间下拉栏时，体验非常糟糕。
- 自定义课程可以单独存表，例如现在我需要添加“晚读”“眼保健操”两个课程，则需要反复输入两个词语；如果可以存表，仅需在自定义的二级目录中选取即可，方便快捷。
- 最右侧的预览课表优化：课程简称需要自定义，否则无法区分“早操”“早读”这类的活动。也需要添加“是否预览下午课程”之类的选项。

无论如何，谢谢您的开发。这是我目前所想到的可行的优化内容，其中部分建议，如果我有时间也会参与到项目的贡献中。

**Comment by RinLit-233-shiroko** [comment]
好，我会尽力实现的

**Comment by Leozhu2010** [comment]
触摸屏适配这个确实急需，我们现在老师很多都说这玩意碍事（主要是我们屏幕4kdpi300%，即使开了上课自动隐藏不会误触点下来也回挡住wps跟希沃白板之类软件的标签页栏跟大部分软件的标题栏）
